common/src/main/java/com/viaversion/viaversion/protocols/v1_20_3to1_20_5/rewriter/BlockItemPacketRewriter1_20_5.java

```diff
@@ -387,6 +387,11 @@ public Item toOldItem(final UserConnection connection, final Item item, final St
             return dataItem;
         }
 
+        // In 1.20.6, some items have default values which are not written into the components
+        if (item.identifier() == 1105 && !data.contains(StructuredDataKey.FIREWORKS)) {
```

Comment:
only if backupInconvertibleData is set in the dataConverter

Reply:
Maybe we should consider renaming the boolean then?

Reply:
Anyway, fixed